src/components/ModalExpel/index.tsx

```diff
@@ -22,7 +22,7 @@ export const ModalExpel: FC<Props> = ({
 }) => {
   const onSubmitModal = () => {
     onClose();
-    if (onSubmit) onSubmit('Modalexpel');
+    if (onSubmit) onSubmit();
```

Comment:
I'd recommend using curly braces for consistency

Reply:
`if (onSubmit) {
    onSubmit();
}`

Reply:
Done